src/Raven.Client/ServerWide/Operations/DatabasesInfo.cs

```diff
@@ -155,7 +158,23 @@ public DynamicJsonValue ToJson()
                 [nameof(DriveName)] = DriveName,
                 [nameof(VolumeLabel)] = VolumeLabel,
                 [nameof(TotalFreeSpaceInBytes)] = TotalFreeSpaceInBytes,
-                [nameof(TotalSizeInBytes)] = TotalSizeInBytes
+                [nameof(TotalSizeInBytes)] = TotalSizeInBytes,
+            };
+        }
+    }
+
+    public class IoStatsResult
+    {
+        public double ReadIOs { get; set; }
+
+        public double WriteIOs { get; set; }
```

Comment:
```suggestion
        public double IoReadOperations { get; set; }

        public double IoWriteOperations{ get; set; }
```

Also we'd also want number of sectors (read and write) (1 sector = 4KB) which gives us throughput.

Also I think `in_flight` value is good to have - it's queue length.
```
in_flight
=========

This value counts the number of I/O requests that have been issued to
the device driver but have not yet completed.  It does not include I/O
requests that are in the queue but not yet issued to the device driver.

```

Reply:
I don't think we can assume that sector is 4KB, can we?

Reply:
We probably can't, but we can check it: https://unix.stackexchange.com/questions/2668/finding-the-sector-size-of-a-partition

Reply:
I can read it directly from the `stats` file
```
Name            units         description
----            -----         -----------
read I/Os       requests      number of read I/Os processed
read merges     requests      number of read I/Os merged with in-queue I/O
read sectors    sectors       number of sectors read
read ticks      milliseconds  total wait time for read requests
write I/Os      requests      number of write I/Os processed
write merges    requests      number of write I/Os merged with in-queue I/O
write sectors   sectors       number of sectors written
```

Reply:
As written in the documentation the section is always 512 byte
```
read sectors, write sectors, discard_sectors
============================================

These values count the number of sectors read from, written to, or
discarded from this block device.  The "sectors" in question are the
standard UNIX 512-byte sectors, not any device- or filesystem-specific
block size.  The counters are incremented when the I/O completes.
```

Reply:
Great, that's even better 👍